### ISSUE TITLE
Contract data load failure

### ISSUE THREAD

**Issue by sadeghte** [muon-protocol/muon-node-js#351]
Sometimes, an error occurs when loading contact data and it does not resolve until the app is restarted. 
The error is:

```
muon:network:utils loading NodeManager data failed. 
ContractExecutionError: Error happened while trying to execute a function inside a smart contract 
at Function._isReverted (file:///usr/src/muon-node-js/node_modules/web3-core/src/web3_request_manager.ts:432:48) 
at Web3RequestManager._processJsonRpcResponse (file:///usr/src/muon-node-js/node_modules/web3-core/src/web3_request_manager.ts:361:35) 
at Web3RequestManager.<anonymous> (file:///usr/src/muon-node-js/node_modules/web3-core/src/web3_request_manager.ts:206:16) 
at Generator.next (<anonymous>) 
at fulfilled (file:///usr/src/muon-node-js/node_modules/web3-core/lib/esm/web3_request_manager.js:20:58) 
at processTicksAndRejections (node:internal/process/task_queues:95:5) 
{ 
	innerError: [Eip838ExecutionError: execution reverted] 
	{ 
		innerError: undefined, 
		code: -32000, 
		receipt: undefined, 
		data: undefined 
	}, 
	code: 310, 
	receipt: undefined 
}
```

**Comment by bakhshandeh** [comment]
@sadeghte if this happens on ALICE, we should add the error to list of the recoverable errors.
One of the RPCs is returning error and it does not switch to a new one.

**Comment by sadeghte** [comment]
fixed in [d2d29f9](https://github.com/muon-protocol/muon-node-js/commit/d2d29f91aa51fec6d3f17a2ba66c9bbb960427d3)
